mm/ktsan/stack.c

```diff
@@ -14,7 +14,6 @@ void kt_stack_print(kt_stack_t *stack, uptr_t top_pc)
 		pc = kt_decompress(stack->pc[i]);
 		pr_err(" [<%p>] %pS\n", (void *)pc, (void *)pc);
 	}
-	pr_err("\n");
```

Comment:
This was done on the purpose, all user-space sanitizers always terminate stacks with new line. This significantly improves report readability, and was suggested by external people who know how to make things look good.